core-features/generate-client-and-cli-install-yarn-lockfile/test.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -eux
+
+if [ -f "pnpm-lock.yaml" ] || [ -f "package-lock.json" ]; then
+  echo "Test should have produced a yarn.lock file only"
+  exit 1
+fi
```

Comment:
before actually running the test?

Reply:
Yes, the test check the contents of the package.json and this looks that only the correct lockfile was created by generate.

Reply:
Ok that's a bit confusing: "Test should have..." while the test is actually run below this code.
Do you mean `run.sh should have ...` maybe?

Reply:
Yes, definitely. I'll clear that part.